plugins/kubernetes/src/components/Pods/PodDrawer/ContainerCard.tsx

```diff
@@ -84,7 +84,7 @@ const ContainerDatetime = ({ prefix, dateTime }: ContainerDatetimeProps) => {
 };
 
 export interface ContainerCardProps {
-  logContext: PodLogContext;
+  logContext: PodScope;
```

Comment:
Feel free to disagree here, but maybe we rename all the instances of `logContext` to be `scope`?

Reply:
fixed